assets/dev/js/editor/elements/views/container.js

```diff
@@ -412,6 +412,12 @@ const ContainerView = BaseElementView.extend( {
 				this.reInitEmptyView();
 			}
 
+			// Todo: Remove in version 3.21.0: https://elementor.atlassian.net/browse/ED-11884.
```

Comment:
Do we have a better way to follow this function removal?
Our codebase is full of TODO's
Can we use the deprecation system?